### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SomparbT**
Update readme with more details of project.

### CODE REVIEW

**Review by CHNUTT — DISMISSED** [review]
Welcome to the project!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+
+![Bugs](https://img.shields.io/github/issues/thai-programmers-usa/thai-programmers-usa.github.io/bug?label=Bugs) ![Feature Requests](https://img.shields.io/github/issues/thai-programmers-usa/thai-programmers-usa.github.io/feature-request?color=blue&label=Feature%20Requests)
+
+## Roadmap
+
+Here's a snippet of what's currently ✅ available or ❌ unavailable and what we have on our radar for the ⏩future:
+
+Current Features| Future Roadmap |
+|:--|:--|
+❌ Payment Transaction <br>  | ⏩ Sign up/ login <br> ⏩ Profile Edit <br> ⏩ Online name cards <br>
```
Comment (BayLadyCoder) [inline README.md:14]: Why Payment Transaction is listed here?

Comment (CHNUTT) [reply README.md:14]: Probably he meant show that no payment required?

Comment (BayLadyCoder) [reply README.md:14]: I think it means `❌ unavailable and what we have on our radar for the`

Comment (BayLadyCoder) [reply README.md:14]: But I don't think we have any plan around that topic yet

Comment (CHNUTT) [reply README.md:14]: You can require the change to block the merge till we got answer

Comment (SomparbT) [reply README.md:14]: Just wanted to put something in there, probably too much misleading.

Comment (SomparbT) [reply README.md:14]: @BayLadyCoder Those two are different sentences.
- ❌ unavailable
- , and what we have on our radar for the ⏩future:

Comment (BayLadyCoder) [reply README.md:14]: I see. Let's remove Payment Transaction for now. We can replace it with other topics that we might add later after this online name cards project is done such as a job board or community board.

Comment (BayLadyCoder) [reply README.md:14]: Or just skip unavailable topic entirely, is fine too.

**Review by BayLadyCoder — APPROVED** [review]

**Review by CHNUTT — APPROVED** [review]

**Pull request merged**